src/ucs/config/global_opts.c

```diff
@@ -29,6 +29,8 @@ ucs_global_opts_t ucs_global_opts = {
     .log_buffer_size       = 1024,
     .log_data_size         = 0,
     .mpool_fifo            = 0,
+    /* docker container has 64m /dev/shm by default, make sure not to exceed it */
+    .mpool_max_chunk_size  = 17 * UCS_MBYTE,
```

Comment:
Please elaborate:
1. Why set a default value according to docker container limitations? It is not the main use-case, we can rather have defaults which are unsuitable for docker, and document needed user adaptations when running in docker.
2. What is the relation between 64MB and 17MB?

Reply:
Will change to a more reasonable value.

Reply:
changed to 32m.

Reply:
1)Why set a default value according to docker container limitations? It is not the main use-case, we can rather have defaults which are unsuitable for docker, and document needed user adaptations when running in docker.
[Answer] Right. Should set a more reasonable value and see if need change related CI cases.
I met below CI error in docker:
mpool "mm_recv_desc" used uct_posix_mem_alloc() with grow_factor 2, 
its 1st chunk_size was 4292720, 2nd chunk_size 8585328, 3rd chunk_size 17170544, then 4th chunk_size 34340976. 
It always failed to alloc 4th chunk in docker because docker had by default 64M /dev/shm, and uct_posix_mem_alloc() would write 34340976 to some file under this partition to test memory via uct_posix_test_mem(). We can see the four total chunk_size would be near 64m.

2)What is the relation between 64MB and 17MB?
[Answer] In fact, for the failed cases, set a value less than 34340976 would work. I chose 17m which was the minimum value to make this pool grow twice, I also thought it was strange and not good at all now.